visualizeInput.py

```diff
@@ -0,0 +1,57 @@
+# To visualise the data from Hacktoberfest_Inputt, we plot the occcurences of 
+# each relation
+
+import pandas as pd
+import networkx as nx
+import matplotlib.pyplot as plt
+from pprint import pprint
+from collections import Counter
+
+
+
+def visualiser(fileName):
+    df = pd.read_excel(fileName).to_dict()
+    # pprint(df)
+    rows = list(df['Source Color'].keys())
+
+    column_A = list(df['Source Color'].values())    #Source colors
+    column_B = list(df['Dest color'].values())      #Destination colors
+
+    # Zone relations
+    relations = []
+
+    # Relations from to column 1 to column 2 for the directed graph
+    relG = {'from':[],'to':[]}
+
+    # Colors available
+    colors=['blue','brown','green','grey','red','orange','yellow']
+
+    for i in rows:
+        if (str(column_A[i])).lower() in colors:
```

Comment:
**NameError: name 'column_A' is not defined**

i am sure for column B as well, I'll get one.

Reply:
Interesting, coz I'm not getting any error like that. I have declared the list for column_A and for B as well. I'm not sure what's causing this error.

Reply:
Is any declaration commented out on your clone?

Reply:
You aren't able to print column_A as well?

Reply:
Sure, I'll pull up a new PR for that😉